jetty-util/src/main/java/org/eclipse/jetty/util/MultiPartInputStreamParser.java

```diff
@@ -642,11 +655,11 @@ else if(tl.startsWith("filename="))
                 InputStream partInput = null;
                 if ("base64".equalsIgnoreCase(contentTransferEncoding))
                 {
-                    partInput = new Base64InputStream((ReadLineInputStream)_in);
+                    partInput = new Base64InputStream(is);
                 }
                 else if ("quoted-printable".equalsIgnoreCase(contentTransferEncoding))
                 {
-                    partInput = new FilterInputStream(_in)
+                    partInput = new FilterInputStream(is)
```

Comment:
I don't think I follow. I used mark when possible as `_in` is already a BufferedInputStream.